app/scripts/metamask-controller.js

```diff
@@ -5351,7 +5298,6 @@ export default class MetamaskController extends EventEmitter {
     }
 
     // Ensure preferences + identities controller know about all addresses
```

Comment:
Nit: Do we need to update this comment? Or even better, nix it (because the code describes what we are doing already)

``` suggestion
```